### DISCUSSION TITLE
Unable to log in to the original account with oauth after changing the email address

### DISCUSSION THREAD

**Discussion by nexmoe** [appwrite/appwrite#3898] [Q&A]
The logic of using oauth to login after changing the email address will re-register a new account, which is not quite right

**Comment by maeddin** [comment]
Thanks for bringing this up. This was the reason for me to migrate my system away from Appwrite. The account management logic didn't make sense to me. But I probably didn't sum up my problem as well as you did at the time. 😅

**Comment by nexmoe** [comment]
If users change their e-mail in providers, what will happen?
I didn't try, but I guess they will lose their account in the apps built with appwrite.

**Reply by maeddin** [reply]
As far as I remember, you can no longer log in with the OAuth methods you used before, but only with the OAuth methods of the new email. This comes from the uniqueness of emails in Appwrite. But I'm not sure anymore.
